public/css/room-directory.css

```diff
@@ -409,3 +429,24 @@
   background-color: #2774c2;
   color: #ffffff;
 }
+
+.RoomDirectoryView_safeSearchInput {
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  -o-appearance: none;
+  appearance: none;
+  position: relative;
+  width: 40px;
+  height: 20px;
+  background-color: #ffffff;
+  border: 1px solid rgba(200, 200, 200, 1);
+  box-shadow: inset -20px 0px 0px 0px rgba(200, 200, 200, 1);
+  border-radius: 10px;
+  transition: all 0.2s ease;
+  right: 0;
+}
+
+.RoomDirectoryView_safeSearchInput:checked {
+  order: 1px solid rgba(80, 134, 240, 1);
+  box-shadow: inset 20px 0px 0px 0px rgba(80, 134, 240, 1);
+}
```

Comment:
We need to run Prettier over the `.css` styles as well

CI failure: https://github.com/matrix-org/matrix-public-archive/actions/runs/3794941206/jobs/6453539585#step:5:10